.github/workflows/publish.yml

```
@@ -10,30 +10,27 @@ on:
 jobs:
   publish:
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: write
     if: github.repository == 'EmpaEconversion/aurora-neware'
 
-
     steps:
-      # Uncomment the following steps to publish to PyPI using flit
-
-      # - name: Checkout code
-      #   uses: actions/checkout@v4
+      - name: Checkout code
+        uses: actions/checkout@v4
 
-      # - name: Set up Python
-      #   uses: actions/setup-python@v5
-      #   with:
-      #     python-version: '3.11'
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
 
-      # - name: Install flit
-      #   run: pip install flit~=3.12
+      - name: Install flit
+        run: pip install flit~=3.12
 
-      # - name: Build and publish
-      #   run: flit publish
-      #   env:
-      #     FLIT_USERNAME: __token__
-      #     FLIT_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+      - name: Publish to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
```

I suggest using `flit publish` as it does everything in one go:

```suggestion
      - name: Build and publish to PyPI
        run: flit publish
```